app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user, foreign_key: :author_id
```

Comment:
- [ ] According to the ERD, the foreign key in this comment table should be AuthorID, therefore the association should be:
`belongs_to :author, class_name: 'User'`

Please consider refactoring this appropriately.